env.mjs

```diff
@@ -3,11 +3,11 @@ import { z } from "zod";
 
 export const env = createEnv({
   server: {
-    TWITTER_CLIENT_ID: z.string().optional(),
-    TWITTER_CLIENT_SECRET: z.string().optional(),
-    IPINFO_TOKEN: z.string(),
-    SCORECHAIN_API_KEY: z.string(),
-    AML_ALERT_WEBHOOK: z.string().url(),
+    // TWITTER_CLIENT_ID: z.string().optional(),
```

Comment:
You can uncomment these envs, instead you can apply optional field to `IPINFO_TOKEN`, `SCORECHAIN_API_KEY`, and `AML_ALERT_WEBHOOK` fields, like the above fields(ie. `TWITTER_CLIENT_ID`).